packages/react-composites/src/composites/CallComposite/ConfigurationScreen.tsx

```diff
@@ -40,7 +40,7 @@ export const ConfigurationScreen = (props: ConfigurationScreenProps): JSX.Elemen
         />
       </div>
       <div>
-        <StartCallButton onClickHandler={startCallHandler} isDisabled={false} />
+        <StartCallButton onClickHandler={startCallHandler} isDisabled={!microphonePermissionGranted} />
```

Comment:
One question about this, do we wanna disable user ability to join a call when they don't have permission for mic?
I would say under some cases, it could be possible that I am joining an online lecture(which I don't need to speak), and I just don't wanna give my mic permission, then I can't join that online meeting

Reply:
Right, that is a possible scenario, I asked David about this and they still want to disable the call button when we don't permission for mic to reduce confusion of being in a call and not being able to speak.